### PR TITLE
Cassandra 3.0.8

### PR DESCRIPTION
**Pull request by cdonati**
This installs an OpenJDK 8 JRE on Precise and Trusty.

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2038/

File: debian/appscale_build.sh [hunk 1]
```diff
@@ -67,6 +67,20 @@ if [ "${DIST}" = "wheezy" ]; then
     curl https://haproxy.debian.net/bernat.debian.org.gpg | apt-key add -
 fi
 
+# Trusty, Precise, and Wheezy do not have Java 8.
+case "$DIST" in
+    precise|trusty) apt-add-repository -y ppa:openjdk-r/ppa ;;
+    wheezy)
+        echo "This script does not automatically install Java 8 on Debian "\
```
Comment (menivaitsi) [inline debian/appscale_build.sh]: Is this for upgrades?

Comment (cdonati) [inline debian/appscale_build.sh]: It will show up for any build on Wheezy.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2050/

**Pull request merged**